lib/widgets/theme.dart

```diff
@@ -111,6 +123,12 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
   // Not exactly from the Figma design, but from Vlad anyway.
   final StreamColorSwatches streamColorSwatches;
 
+  // Not named variables in Figma; taken from older Figma drafts, or elsewhere.
+  final Color star;
+  final Color editedMovedMarkerBg;
+  final Color editedMovedMarkerExpanded;
```

Comment:
These two aren't used yet in this commit (or PR), right? Let's save them for the commit where they do get used:
```suggestion
```